tests/puppeteer/campaigns/functional/BO/07_modules/01_moduleManager/02_filterModulesByCategories.js

```diff
@@ -0,0 +1,69 @@
+require('module-alias/register');
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const ModuleManagerPage = require('@pages/BO/modules/moduleManager');
+
+let browser;
+let page;
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    moduleManagerPage: new ModuleManagerPage(page),
+  };
+};
+
+describe('Filter modules by Categories', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+  });
+  // Login into BO and go to module manager page
+  loginCommon.loginBO();
+
+  it('should go to module manager page', async function () {
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.modulesParentLink,
+      this.pageObjects.boBasePage.moduleManagerLink,
+    );
+    await this.pageObjects.boBasePage.closeSfToolBar();
+    const pageTitle = await this.pageObjects.moduleManagerPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.moduleManagerPage.pageTitle);
+  });
+
+  describe('Filter modules by categories', async () => {
+    [
+      'Administration',
```

Comment:
You might want to keep this list in some config file as they can change if Product team decides to rename them ?